### ISSUE TITLE
Improve modularization of code

### ISSUE THREAD

**Issue by caldwellst** [OCHA-DAP/pa-aa-toolbox#101]
Given that we expect to continually expand our data sources, I think the namespace of the library is going to be increasingly crowded. Was wondering if for the next release after `0.4.0` we should aim to improve the module organization, so rather than evertyhing being at the top level, having a clear design to follow. Not sure what that would look like, but something like our folder structure, so you'd have to go to specific data sources to get those data source classes, for instance.

Anyway, I tagged this myself, feel free to edit/remove, and interested to see thoughts!

**Comment by turnerm** [comment]
Agreed, and @caldwellst would you say it's related to #42?

**Comment by Tinkaa** [comment]
I agree with both issues, but I would say these are different though intertwined. This one is imo on how we define the structure of importing classes whereas #42 is about the folder structure.

**Comment by turnerm** [comment]
Let's think about them both for the next release
